File: src/main/java/pro/sky/java/LinksShortenerApplication.java
package pro.sky.java;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableAsync
@EnableScheduling
@SpringBootApplication
public class LinksShortenerApplication {

    public static void main(String[] args) {
        SpringApplication.run(LinksShortenerApplication.class, args);
    }
}

File: src/main/kotlin/com/github/disterru/linksshortener/LinksShortenerApplication.kt
package com.github.disterru.linksshortener

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class LinksShortenerApplication

fun main(args: Array<String>) {
    runApplication<LinksShortenerApplication>(*args)
}